apps/web/src/app/admin-console/common/people-table-data-source.ts

```diff
@@ -213,3 +241,130 @@ export abstract class PeopleTableDataSource<T extends UserViewTypes> extends Tab
     }
   }
 }
+
+export class ProvidersTableDataSource extends PeopleTableDataSource<ProviderUser> {
+  protected statusType = ProviderUserStatusType;
+}
+
+export class MembersTableDataSource extends PeopleTableDataSource<OrganizationUserView> {
+  protected statusType = OrganizationUserStatusType;
+}
+
+export interface BulkFlags {
+  showConfirmUsers: boolean;
+  showBulkConfirmUsers: boolean;
+  showBulkReinviteUsers: boolean;
+}
+
+export interface BulkMemberFlags extends BulkFlags {
+  showBulkRestoreUsers: boolean;
+  showBulkRevokeUsers: boolean;
+  showBulkRemoveUsers: boolean;
+  showBulkDeleteUsers: boolean;
+}
+
+/**
+ * Helper function to determine if the confirm users option should be shown
```

Comment:
To be more specific, I think this is a callout / banner, not the menu option.

```suggestion
 * Helper function to determine if the confirm users callout should be shown
```